Makefile

```diff
@@ -26,11 +36,11 @@ core/js/core.bundle.js: node_modules
 # core
 #
 .PHONY: core
-core: core/js/core.bundle.js core/css/uikit.owncloud.css
+core: core/js/core.bundle.js core/css/uikit.owncloud.css core/css/material-icons.css core/fonts/MaterialIcons-Regular.%
```

Comment:
this might work only by chance

the "%" placeholder here will never have a value

I suggest just using "core/fonts" as dependency and have all the copying stuff done inside the "core/fonts" rule